DP02_13a_Image_Cutout_SciDemo.ipynb

```diff
@@ -0,0 +1,1286 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #53.        f"Datalink link service url: {dataLinkUrl}"
</pre><p>This isn't printing anything?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Interesting. It seems like this only prints (permanently?) if its the last line in the cell. </p>

Reply:
<p>I have moved all of the f" " calls to the last of the cell</p>